modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -1082,6 +1082,18 @@ public void postProcessModelProperty(CodegenModel model, CodegenProperty propert
             }
         }
 
+        // add implements for serializable/parcelable to all models
+        List<Object> models = (List<Object>) objs.get("models");
+        for (Object _mo : models) {
+            Map<String, Object> mo = (Map<String, Object>) _mo;
```

Comment:
Why are you casting twice instead of only once:

```suggestion
        List<Map<String, Object>> models = (List<Map<String, Object>>) objs.get("models");
        for (Map<String, Object> mo : models) {
```